Formula/minio.rb

```diff
@@ -2,9 +2,9 @@ class Minio < Formula
   desc "Amazon S3 compatible object storage server"
   homepage "https://github.com/minio/minio"
   url "https://github.com/minio/minio.git",
-    :tag => "RELEASE.2016-12-13T17-19-42Z",
-    :revision => "29d72b84c07f9555f83a6485fe8291e18d23811b"
-  version "20161213171942"
+    :tag => "RELEASE.2017-01-25T03-14-52Z",
+    :revision => "f8e4700a11065967242b8857045ce7dee607722d"
```

Comment:
Looks like this may be wrong: https://bot.brew.sh/job/Homebrew%20Core/15284/version=sierra/testReport/junit/brew-test-bot/sierra/fetch_minio/